server/src/main/java/dev/slimevr/vr/trackers/IMUTracker.java

```diff
@@ -396,6 +474,11 @@ public boolean hasPosition() {
 		return false;
 	}
 
+	@Override
+	public boolean hasAcceleration() {
+		return true;
```

Comment:
What if an IMU doesn't have accel data? Would that cause any problem, or would the accel vector being (0, 0, 0) suffice?

Reply:
With new firmware acceleration is always sent, unless explicitly disabled by the user in defines. For trackers with old firmware this will be a zero vector, yes.

Reply:
Alright